src/app/series/series.component.ts

```diff
@@ -90,6 +90,12 @@ export class SeriesComponent implements OnInit {
       if (wasNew) {
         this.router.navigate(['/series', this.series.id]);
       }
+    },
+    err => {
```

Comment:
This goes with that questionable 500 error message handling that I was attempting. There's a styleguide branch dependent on it out there too: https://github.com/PRX/styleguide.prx.org/commit/e429c7ce6ff45ddaccfe0cbbb3ae5831fd5f4061 Do you think we should keep it? It felt pretty wrong with the res['_body'] and the JSON.parse()